build-scan-publish-maven/action.yml

```diff
@@ -18,6 +18,9 @@ inputs:
   skip-summary:
     description: 'Whether to add or not a summary to the GitHub workflow'
     default: 'false'
+  skip-project-id:
```

Comment:
I think this should be called `skip-project-id-in-summary` or `exclude-summary-project-id` (or similar). Here in `action.yml` it makes sense due to context, but in a workflow file it's not clear what `skip-project-id` would do.

Reply:
Makes sense, I changed it to `skip-project-id-in-job-summary`